Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```
@@ -138,8 +138,8 @@
   name: Enforcer
   parent: [BaseWeaponShotgun, BaseGunWieldable, BaseRestrictedContraband]
   id: WeaponShotgunEnforcer
-  description: A premium combat shotgun based on the Kammerer design, featuring an upgraded clip capacity. .50 shotgun shells.
-  components:
+  description: A premium semi-auto combat shotgun used by modern security forces. Uses .50 shotgun shells.
```

I prefer auto being expanded to automatic (more consistency). Also, what are "_modern_ security forces""

```suggestion
  description: A premium semi-automatic combat shotgun, which is the pride of all security forces. Uses .50 shotgun shells.
```